travis/scripts/04-tests.sh

```diff
@@ -22,9 +22,13 @@ fi
 #-------------------------------------------------------------------------------
 cd "$HOME"/app
 if [ -f "mvnw" ]; then
-    ./mvnw test -Dspring.jpa.show-sql=false
+    ./mvnw test \
+        -Dlogging.level.io.github.jhipster.sample=INFO \
```

Comment:
I think we could even set to ERROR as thats what we would be interested in wont we?